scenario/prechecks.yaml

```diff
@@ -28,3 +28,4 @@ preparation:
           - check_dst_sql: True
           - check_dst_rabbit: True
       - check_networks: True
+  - check_users_availability: True
```

Comment:
It seems we have 2 the same scenarios: this one and `scenario/stages/0_prechecks.yaml`. Maybe we'll stay only with one?